<?xml version="1.0"?>
<launch>
	<rosparam file="$(find evo_ros2)/config/evo_ros2_config.yaml" command="load"/>
	<rosparam file="$(find evo_ros2)/config/genome_mapping.yaml" command="load"/>
	
	<node name = "transporter_node" pkg="evo_ros2" type="transporter_node.py" output="screen"/>
	<node name = "software_manager_node" pkg="evo_ros2" type="software_manager_node.py" output="screen"/>
	
	<!-- <node name = "evo_ros2_soft_reset_service_node" pkg="evo_ros2" type="reset_sim.py" output="screen"/> -->

</launch>
